### ISSUE TITLE
Installed python package does not find log4cplus config

### ISSUE THREAD

**Issue by MaxNoe** [tudo-astroparticlephysics/PROPOSAL#86]
```
log4cplus:ERROR could not open file /tmp/pip-req-build-79oqjkru/resources/log4cplus.conf
log4cplus:ERROR No appenders could be found for logger (PROPOSAL).
log4cplus:ERROR Please initialize the log4cplus system properly.
```

The path is the temporary build path.

Any ideas how to adapt this, so it can find a path at installation?

**Comment by MaxNoe** [comment]
On closer inspection, this file should not be used at all. It looks like an example file, not something that should be used by default.

Let's define the default behaviour in code and give an example how to set the logfile to a user provided one?

**Comment by sudojan** [comment]
Can you point out where this error occurs?

**Comment by MaxNoe** [comment]
If you start any python script using PROPOSAL and PROPOSAL tries to log something.

I fixed it in #87

**Comment by MaxNoe** [comment]
```
$ BUILD_CORES=12 pip install --user .
$ python resources/examples/standalone/plot_continuous.py 
log4cplus:ERROR could not open file /tmp/pip-req-build-glt6hfae/resources/log4cplus.conf
log4cplus:ERROR No appenders could be found for logger (PROPOSAL).
log4cplus:ERROR Please initialize the log4cplus system properly.
```

**Comment by Jean1995** [comment]
> If you start any python script using PROPOSAL and PROPOSAL tries to log something.
> 
> I fixed it in #87

I was able to reproduce this error when trying to run an example script

**Comment by MaxNoe** [comment]
After reading the log4cplus docs on how to initialize the logging system, this is anyways necessary.

And setting the logfile at runtime via en environment variable is a nice feature imho
